static/carousel/css/styles.css

```diff
@@ -47,7 +47,6 @@ body  {
   right: 0;
   position: absolute;
   margin: auto;
-  -webkit-transition: all .2s ease;
```

Comment:
Just wondering if this was this intentional:
To remove `-webkit-` vendor prefix

I'm not sure but it's probably safe to remove it, please confirm, thanks.